queries/allCountries.sql

```diff
@@ -0,0 +1,11 @@
+# All countries in the world organised by largest population to smallest.
+SELECT code, name, continent, region, population, (
+    SELECT name
+    FROM city ci
+    WHERE countrycode = co.code
+        AND ci.id = co.capital
+    ) AS capital
+FROM country co
+ORDER BY population DESC;
+
```

Comment:
it'd probably be good to include the queries with the variables like in the other sql files, but i've got a better designed country query that i'm going to replace these with, so it's fine to leave it for now.